rs/state_machine_tests/src/lib.rs

```diff
@@ -567,12 +574,13 @@ impl PocketIngressPool {
             validated: btreemap![],
         }
     }
+
     /// Pushes a received ingress message into the pool.
     fn push(&mut self, m: SignedIngress, timestamp: Time) {
         self.validated.insert(
             IngressMessageId::new(m.expiry_time(), m.id()),
             ValidatedIngressArtifact {
-                msg: m.into(),
+                msg: IngressPoolObject::new(NODE_1, m),
```

Comment:
Why not use the node ID of the first node on the subnet modeled by this StateMachine, e.g., something like `self.nodes[0].node_id` instead of `NODE_1`?

Reply:
Good point, thanks!
Done.